packages/cubejs-server-core/core/index.d.ts

```diff
@@ -100,7 +100,7 @@ declare module "@cubejs-backend/server-core" {
     | "sqlite";
 
   export interface QueryFilter {
-    member: string;
+    dimension: string;
```

Comment:
`member` and `dimension` are both allowed however `dimension` is deprecated here.